example/lambdas/python-processing/process_activity.py

```diff
@@ -0,0 +1,74 @@
+"""Reference python activity implementaion
+
+Creates a stub cumulus-process-py stub implementation that
+takes a payload and returns a mocked output for test purposes
+"""
+
+import copy
+import os
+
+from hashlib import md5
+
+from cumulus_process import Process
+from cumulus_process.s3 import upload
+
+# pylint: disable=R0201
+class PythonProcess(Process):
+    """
+        A subclass of Process for creating .md5 hash files for test Modis granules
+        This is an example and does not intended for processing MODIS Products
+    """
+    @property
+    def input_keys(self):
+        """ This property is helps the processing step to distinguish incoming
+            files from one another
+        """
+        return {
+            'hdf': r"^.*\.hdf$",
+            'all': r".*"
+        }
+
+    def process(self):
+        granules = copy.deepcopy(self.input['granules'])
+        # We have to reasssgn the class input each time in an activity
+        # as the library doesn't appear to handle download *and* the service
+        # case.   This should be fixed.
+        original_file_names = [file['filename'] for granule in granules
```

Comment:
TIL: Python is wild 🐍

Reply:
One of the things I really dislike about node is it doesn't have list/dictionary comprehensions.